bears/python/requirements/PySafetyBear.py

```diff
@@ -74,12 +74,20 @@ def run(self, filename, file):
                 version_spec_match.end(1) + 1,
             )
 
-            yield Result(
-                self,
-                message_template.format(vuln=vulnerability),
-                additional_info=vulnerability.description,
-                affected_code=(source_range, ),
-            )
+            try:
+                yield Result(
+                    self,
+                    message_template.format(vuln=vulnerability),
+                    additional_info=vulnerability.description,
+                    affected_code=(source_range, ),
+                )
+            except:  # Ignore PycodestyleBear (E722)
```

Comment:
Please add test case to cover this scenario as well. That's why travis CI is failing due to coverage.

Reply:
Added